content/documentation/edge/quick-start/index.adoc

```diff
@@ -57,16 +94,30 @@ istioctl dashboard kiali
 
 icon:bullhorn[size=1x]{nbsp} The default credentials to log in are: `admin/admin`. It is highly recommended to change them as soon as possible, see the link:{{< ref "/documentation/edge/installation-guide#_login_options" >}}[Login Options]
 
-=== Running on OpenShift
+==== Running on OpenShift
 
-When Kiali runs on OpenShift it needs access to some OpenShift specific resources in order to function properly, which can be done using the following commands after Kiali has been installed:
+When installing using `istioctl` on OpenShift, Kiali needs to be given access to some OpenShift specific resources in order to function properly, which can be done using the following commands after Kiali has been installed:
 
 ----
 $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiGroups":["apps.openshift.io"], "resources":["deploymentconfigs"],"verbs": ["get", "list", "watch"]}}]' --type json
 $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiGroups":["project.openshift.io"], "resources":["projects"],"verbs": ["get"]}}]' --type json
 $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiGroups":["route.openshift.io"], "resources":["routes"],"verbs": ["get"]}}]' --type json
 ----
 
+== Open the UI
+
+Once the Kiali Server has been installed and has successfully started, you can access the UI. For more details, check the link:{{< ref "/documentation/edge/faq/general#how-do-i-access-kiai" >}}[FAQ: How do I access Kiali UI?]
+
+The Istio `istioctl` client has an easy method to access the Kiali UI:
```

Comment:
Again just asking that this is still gonna be there...

Reply:
Yes, I think so.